src/handlers/notify.rs

```diff
@@ -134,13 +131,10 @@ async fn process_publish_jobs(
                 response.sent.insert(account.to_string());
```

Comment:
Need the topic and ideally `rpc_id` of the relay message here

Reply:
Added topic. `rpc_id` may be problematic as `relay client` is not sharing this data with consumer.
I'm currently working on moving from this to HTTP client. Will try to fix it with it if that's ok

Reply:
Yup cool to add later.